src/rocm_docs/rocm_docs_theme/static/images/alpha-watermark.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="394.000000pt" height="77.000000pt" viewBox="0 0 394.000000 77.000000"
+ preserveAspectRatio="xMidYMid meet">
+
+<g transform="translate(0.000000,77.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
```

Comment:
```suggestion
fill="#a0a0a0" stroke="none">
```

Tweaking this to mid-gray and adjusting the opacity to 20% makes the watermark also visible on the dark theme.

Reply:
Looks good. Changes made in [7ccebb9](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/7ccebb97e512d4eac98eefcae0cc52d5e38b668d)